ecosystem/typescript/sdk/examples/typescript/transactions_management.ts

```diff
@@ -0,0 +1,129 @@
+/**
+ * This example demonstrates how a client can utilize the TransactionWorker class.
+ */
+
+import { AptosAccount, BCS, TxnBuilderTypes, TransactionWorker, FaucetClient, Provider, Types } from "aptos";
+import { exit } from "process";
+import { NODE_URL, FAUCET_URL } from "./common";
+
+const provider = new Provider({ fullnodeUrl: NODE_URL, indexerUrl: NODE_URL });
+
+const faucet = new FaucetClient(NODE_URL, FAUCET_URL);
+
+async function main() {
+  const accountsCount = 5;
+  const transactionsCount = 100;
+  const totalTransactions = accountsCount * transactionsCount;
+
+  console.log("starting...");
+  console.log(new Date().toTimeString());
+  // create senders and recipients accounts
+  const senders: AptosAccount[] = [];
+  const recipients: AptosAccount[] = [];
+  for (let i = 0; i < accountsCount; i++) {
+    senders.push(new AptosAccount());
+    recipients.push(new AptosAccount());
+  }
+  console.log(`${senders.length * 2} sender and recipient accounts created`);
```

Comment:
```suggestion
  console.log(`${senders.length + recipients.length} sender and recipient accounts created`);
```

I think this make more readible instead of *2